README.md

```diff
@@ -0,0 +1,25 @@
+# gz-mujoco converter tool
```

Comment:
The top level directory gz-mujoco is meant to hold various things related to 
Mujoco and Gazebo. The SDFormat->MJCF conversion tool should be a subdirectory. 
See https://github.com/ignitionrobotics/gz-mujoco/pull/2.

Reply:
[a323322](https://github.com/ignitionrobotics/gz-mujoco/pull/3/commits/a323322f54bbf8ac897ae88ff85b7422ee1afcd5)